mitxpro/settings.py

```diff
@@ -498,6 +496,14 @@
     "EMAIL": {"password_reset": "authentication.views.CustomPasswordResetEmail"},
 }
 
+MITXPRO_OAUTH_PROVIDER = "mitxpro-oauth2"
+MITXPRO_OAUTH_APP_NAME = get_string("MITXPRO_OAUTH_APP_NAME", "edx-oauth-app")
```

Comment:
I think this should actually have the `OPENEDX_` prefix, this it's openedx's application within xpro